docs/_includes/demo-page-components/shell.html

```diff
@@ -0,0 +1,284 @@
+<div class="fd-shell fd-shell--fixed fd-shell--fundamentals">
+
+        <div class="fd-shell__header">
+
+
+<div class="fd-shellbar">
+    <div class="fd-shellbar__group fd-shellbar__group--start">
+
+  <div class="fd-shellbar__logo fd-shellbar__logo--image-replaced" aria-label="SAP">
+  </div>
+
+        <div class="fd-shellbar__product">
+
+
+<div class="fd-product-menu">
+  <div class="fd-popover fd-popover--right">
+  <div class="fd-popover__control"><button class="fd-product-menu__control" aria-controls="BYWaF337" aria-haspopup="true" aria-expanded="false">
+
+  <div class="fd-shellbar__logo fd-shellbar__logo--control fd-shellbar__logo--image-replaced fd-product-menu__logo" aria-label="">
+  </div>
+   <span class="fd-shellbar__title fd-product-menu__title">Corporate Portal</span>
+    </button></div>
+
+  <div class="fd-popover__body fd-popover__body--right" aria-hidden="true" id="BYWaF337">
+
+<nav class="fd-menu">
+    <ul class="fd-menu__list">
+      <li><a role="button" class="fd-menu__item">Application A</a>
+    </li>
+      <li><a role="button" class="fd-menu__item">Application B</a>
+    </li>
+      <li><a role="button" class="fd-menu__item">Application C</a>
+    </li>
+      <li><a role="button" class="fd-menu__item">Application D</a>
+    </li>
+    </ul>
+</nav>
```

Comment:
Indentation is off?
I am using beautify plugin in vscode and it formats the html nicely with one shortcut. 
https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify

Reply:
fixed